### PR TITLE
Update cherry-picking-a-commit-in-github-desktop.md

### PR DESCRIPTION
**Pull request by Elanchezhiyan-P**
### Why:

Closes: 

Only the readme file has been updated. After dropping the commit to the respective branch, there is another step to complete the cherry pickup process. You will need to push those changes to the target branch.

### CODE REVIEW

**Comment by cmwilson21** [comment]
@PElanchezhiyan Thanks so much for submitting a PR! I'll get this triaged for review :zap:

File: content/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/cherry-picking-a-commit-in-github-desktop.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ You can also use cherry-picking when collaborating with a team. Some projects in
 
    ![Screenshot of the "History" tab and the "Current Branch" dropdown view. The cursor hovers over the "my-feature" branch, and "plus one" icons indicate the addition of one commit.](/assets/images/help/desktop/cherry-picking.png)
 
+2. After dropping to the respective branch, you can notice that the current branch will be changed to the targetted branch. Then, you need to push the changes to the target branch to see the changes.
```
Comment (isaacmbrown) [inline content/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/cherry-picking-a-commit-in-github-desktop.md:43]: I suggest a few simplifications and editorial updates

```suggestion
1. The current branch changes to the branch onto which you cherry-picked the commit. You can now push the cherry-picked commit to the remote repository.
```

**Review by isaacmbrown — DISMISSED** [review]
Hi @PElanchezhiyan, this is a good point to include, thanks for proposing the update! I suggested a few editorial updates if you wouldn't mind applying those?

**Review by isaacmbrown — APPROVED** [review]
Applied a couple of suggestions and fixed the conflict, thanks for the PR @PElanchezhiyan! 🚀

**Pull request merged**